src/otx/core/data/dataset/base.py

```diff
@@ -159,7 +159,7 @@ def _get_img_data_and_shape(self, img: Image) -> tuple[np.ndarray, tuple[int, in
         key = img.path if isinstance(img, ImageFromFile) else id(img)
 
         if (img_data := self.mem_cache_handler.get(key=key)[0]) is not None:
-            return img_data, img_data.shape[:2]
+            return img_data.astype(np.float32), img_data.shape[:2]
```

Comment:
What's the reason for this change? It is not desired to change the dtype from `uint8` to `float32`. This is because it affects severely to the following image transformation performance.

Reply:
Thanks for the comment. I revert the change.